cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc

```diff
@@ -6,7 +6,7 @@ include::modules/gitops-document-attributes.adoc[]
 
 toc::[]
 
-Once the Red Hat OpenShift GitOps Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO. You can use Keycloak or a bundled Dex OIDC provider.
+Once the Red Hat OpenShift GitOps Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO.
```

Comment:
Should we mention that Dex is unsupported here, or it's already captured elsewhere?

Reply:
@rgarg1 I do not see any mention of Dex supported/not supported anywhere else.

@wtam2018 Should we mention that Dex is unsupported here, WDYT? Thanks!

Reply:
@siamaksade @sbose78 can you let us know your thoughts on this, please?

Reply:
I have we should mention that Dex is unsupported.

Reply:
Done.